src/local_sensitivity/backsolve_adjoint.jl

```diff
@@ -72,7 +72,7 @@ function (S::ODEBacksolveSensitivityFunction)(du,u,p,t)
 
   dλ .*= -one(eltype(λ))
 
-  discrete || accumulate_dgdu!(dλ, y, p, t, S)
+  discrete || accumulate_dgdu!(dλ, y, p, t, S, dgrad)
```

Comment:
we need this in all of them though?

Reply:
Yeah, I only fixed the continuous case. Since you are more familiar with the AD code, do you want to take it from here?

Reply:
I meant, the same change is all that's needed for interpolatingadjoint, right?

Reply:
and something very similar quadratureadjoint, so let's just do all of the continuous cases here.